bolt-http4k/src/main/java/com/slack/api/bolt/http4k/Http4kSlackApp.java

```diff
@@ -1,4 +1,4 @@
-package test_locally;
```

Comment:
@daviddenton This test package name is a convention in this project. Can you revert this change?

Reply:
ah, no. you're right. thanks!